Examples/Examples/SearchExampleView.swift

```diff
@@ -18,7 +18,7 @@ import ArcGISToolkit
 
 struct SearchExampleView: View {
     /// Provides search behavior customization.
-    let locatorDataSource = SmartLocatorSearchSource(
+    @State private var locatorSearchSource = SmartLocatorSearchSource(
```

Comment:
Making this a state property would prevent it from being recreated _if_ this were in a view that was often recomputed. The new name also more closely matches its type.